rest-api-spec/src/main/resources/rest-api-spec/api/fleet.get_secret.json

```
@@ -0,0 +1,28 @@
+{
+  "fleet.get_secret": {
+    "documentation": {
+      "url": null,
+      "description": "Retrieves a secret stored by Fleet."
+    },
+    "stability": "experimental",
+    "visibility":"private",
+    "headers":{
+      "accept": [ "application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_fleet/secrets/{id}",
```

The [ES API conventions](https://docs.elastic.dev/elasticsearch-team/api-guidelines/naming#choosing-the-request-path) recommend that this should be singular.
```suggestion
          "path":"/_fleet/secret/{id}",
```